test/e2e-go/features/transactions/group_test.go

```diff
@@ -202,3 +203,80 @@ func TestGroupTransactionsDifferentSizes(t *testing.T) {
 		a.Error(err)
 	}
 }
+
+func TestGroupTransactionsSubmission(t *testing.T) {
+	t.Parallel()
+	a := require.New(t)
+
+	var fixture fixtures.RestClientFixture
+	fixture.Setup(t, filepath.Join("nettemplates", "TwoNodes50Each.json"))
+	defer fixture.Shutdown()
+
+	client := fixture.LibGoalClient
+	accountList, err := fixture.GetWalletsSortedByBalance()
+	a.NoError(err)
+	account0 := accountList[0].Address
+	goodGroupSizes := []int{1, 2, 16}
+	exceedGroupSize := 17
+
+	sampleTxn, err := client.ConstructPayment(account0, account0, 0, uint64(1000), nil, "", [32]byte{}, 0, 0)
+	a.NoError(err)
+	wh, err := client.GetUnencryptedWalletHandle()
+	a.NoError(err)
+	sampleStxn, err := client.SignTransactionWithWallet(wh, nil, sampleTxn)
+	a.NoError(err)
+
+	for _, gs := range goodGroupSizes {
+		// Generate gs accounts
+		var accts []string
+		for i := 0; i < gs; i++ {
+			acct, err := client.GenerateAddress(wh)
+			a.NoError(err)
+			accts = append(accts, acct)
+		}
+
+		// construct gx txns sending money from account0 to each account
+		var txns []transactions.Transaction
+		for i, acct := range accts {
+			txn, err := client.ConstructPayment(account0, acct, 0, uint64((i+1)*1000000), nil, "", [32]byte{}, 0, 0)
+			a.NoError(err)
+			txns = append(txns, txn)
+		}
+
+		// compute gid
+		gid, err := client.GroupID(txns)
+		a.NoError(err)
+
+		// fill in gid and sign and keep track of txids
+		var stxns []transactions.SignedTxn
+		txids := make(map[string]string)
```

Comment:
why do you need to use `txids` ? I don't see that you're using it.

Reply:
copy-paste of signing code from above, fixed in both places.